keybert/__init__.py

```diff
@@ -1,4 +1,5 @@
 from keybert._llm import KeyLLM
 from keybert._model import KeyBERT
+from keybert.backend import SentenceTransformerBackend
```

Comment:
You can remove this as the backend should be imported as follows:

```python
from keybert.backend import SentenceTransformerBackend
```

and not:

```python
from keybert import SentenceTransformerBackend
```